common/djangoapps/course_modes/models.py

```diff
@@ -359,6 +362,28 @@ def min_course_price_for_currency(cls, course_id, currency):
         modes = cls.modes_for_course(course_id)
         return min(mode.min_price for mode in modes if mode.currency == currency)
 
+    @staticmethod
+    def _create_tuple_from_model(course_mode):
```

Comment:
This might be cleaner as an instance method on the model itself.  Something like `mode.to_tuple()`?